data/relex-semantic.algs

```diff
@@ -772,6 +810,28 @@ HEAD_TO_CLAUSE_AND_WALL
 =
 <F_L head-word> += <F_R head-word>
 ;
+
+#TemplateActionAlg
+NOUN_PHRASE_HEAD_WORD
+<LAB> = \D\.*
+<F_R head-word> = %
+=
+<F_R head-word> = <F_R ref>
+
+#TemplateActionAlg
+DETERMINER_HEAD_WORD
+<LAB> = \D\.*
+<F_L head-word> = %
+=
+<F_L head-word> = <F_R head-word>

```

Comment:
???  This seems deeply mysterious to me ... if you see a determiner, e.g. "the", and the noun on its right has a head-word, then you're going to set the determiners head-word to that?? What for? why would you want to do that?